### PR TITLE
Draft for contributing.md

### PR DESCRIPTION
**Pull request by mjhajharia**
as per #15

cc: @reshamas

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Contributing Guidelines
+
+PyMCON is a community led event and we encourage you to contribute in any form you demarcate suitable for yourself.
```
Comment (reshamas) [inline CONTRIBUTING.md:3]:
```suggestion
PyMCon is a community led event and we encourage you to contribute in any form you demarcate suitable for yourself.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+### Create an issue
+
+If there is something in the website code or the repository that you think can be:
+- Improved
+- Corrected
+- Changed
+
+Then create an issue.
```
Comment (canyon289) [inline CONTRIBUTING.md:32]: Would be nice to link straight to issue button for convenience

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Personal Feedback
+
+Is there something you want to help with, and really think it should be discussed in private? Email the organizers.
```
Comment (canyon289) [inline CONTRIBUTING.md:36]: Add email for easy references

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+### Personal Feedback
+
+Is there something you want to help with, and really think it should be discussed in private? Email the organizers. 
+Please use your own judgement to decide what is appropriate. 
+A few examples of reasonable emails are - accessibility issues you want to discuss in detail personally.
+While this is possible, we strongly encourage opening public issues, so everybody sees them and we can act on it sooner.
+
+## Note
+
+- Please follow the [Code of Conduct](https://pymcon.com/codeofconduct/) throughout.
```
Comment (canyon289) [inline CONTRIBUTING.md:43]: Well put

File: CONTRIBUTING.md (hunk 3 above)
Comment (reshamas) [inline CONTRIBUTING.md:36]:
```suggestion
Is there something you want to help with, and really think it should be discussed in private? [Email the organizers](https://pymcon.com/contact/). 
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  describe the relevant problem in detail with possible links to file or output information
+
+## Resources
+
+If you are not familiar with git, and/or hugo: 
+
+- For making a PR we recommend checking the [Github Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+- For Hugo, we recommend checking [Hugo Docs](https://gohugo.io/documentation/)
+
+Ofcourse you can always ask questions in the Issue or PR itself as well!
```
Comment (reshamas) [inline CONTRIBUTING.md:56]:
```suggestion
Of course you can always ask questions in the Issue or PR itself as well!
```

**Comment by canyon289** [comment]
This is looking good already. Let me know when youre ready for a full review and merge

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+# Contributing Guidelines
+
+PyMCON is a community led event and we encourage you to contribute in any form you demarcate suitable for yourself.
+
+Here are a few possible ways you can contribute:
+
+- [Make a Pull request](###-Make-a-Pull-Request)
+- [Create an Issue](###-Create-an-Issue)
+- [Personal Feedback](###-Personal-Feedback)
+
```
Comment (reshamas) [inline CONTRIBUTING.md:10]:
```suggestion
Here are a few possible ways you can contribute to PyMCon:

- [Make a Pull request](###-Make-a-Pull-Request) to update the website.
- [Create an Issue](###-Create-an-Issue)
- [Personal Feedback](###-Personal-Feedback)
- Share the events with your network

```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+### Make a Pull request
```
Comment (reshamas) [inline CONTRIBUTING.md:12]:
```suggestion
### Make a Pull Request
```

File: CONTRIBUTING.md (hunk 6 above)
Comment (mjhajharia) [reply CONTRIBUTING.md:10]: ah yes this is a good idea

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+If there is something in the website code or the repository that you think can be:
+- Improved
+- Corrected
+- Changed
+
+Then [create an issue](https://github.com/pymc-devs/pymcon_web_series/issues/new/choose).
+
+### Personal Feedback
+
+Is there something you want to help with, and really think it should be discussed in private? Email the organizers.
```
Comment (reshamas) [inline CONTRIBUTING.md]:
```suggestion
Is there something you want to help with, and really think it should be discussed in private? Email the organizers: pymcon@pymc-devs.org 
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
+# Contributing Guidelines
+
+PyMCon is a community led event and we encourage you to contribute in any form you demarcate suitable for yourself.
+
+Here are a few possible ways you can contribute to PyMCon:
+
+- [Make a Pull request](###-Make-a-Pull-Request) to update the website.
```
Comment (reshamas) [inline CONTRIBUTING.md:7]:
```suggestion
- [Make a Pull request](###-Make-a-Pull-Request) (PR) to update the website.
```

**Comment by reshamas** [comment]
@mjhajharia 
Contributing Guide looks great!

**Comment by mjhajharia** [comment]
ok thanks for the reviews @reshamas and @canyon289 this was very useful. Now, quick question, do we want to give the email of the organizers or point them to the "contact organizers" webpage?

**Comment by canyon289** [comment]
contact organizers page so we don't need to keep two things up to date

**Comment by mjhajharia** [comment]
> contact organizers page so we don't need to keep two things up to date

ok! all good then. additionally, not sure how important of a risk this is, but i'd not recommend putting the email address as is in the website. as in we could rather write

pymcon`at`pymc-devs`dot`org 

to reduce the chances of being scraped by spam email senders

**Comment by canyon289** [comment]
@mjhajharia Any issues if we merge this? Its looking real good already

**Comment by mjhajharia** [comment]
> @mjhajharia Any issues if we merge this? Its looking real good already

we're good to go!

**Pull request merged**